firebase-appdistribution-api/firebase-appdistribution-api.gradle

```diff
@@ -47,6 +47,7 @@ dependencies {
     testImplementation "com.google.truth:truth:$googleTruthVersion"
     testImplementation 'androidx.test:core:1.2.0'
     testImplementation 'org.mockito:mockito-inline:3.4.0'
+    testImplementation project(path: ':firebase-appdistribution-api')
```

Comment:
is `path:` necessary?

Reply:
Good catch! Not at all, since its the first param. Removed.